mungegithub/submit-queue/deployment/kubernetes/configmap.yaml

```diff
@@ -91,7 +91,7 @@ nonblocking-jobs: "\
   ci-kubernetes-soak-gke-test,\
   ci-kubernetes-integration-master,\
   ci-kubernetes-verify-master"
-do-not-merge-milestones: ""
+do-not-merge-milestones: "v1.4,v1.5,v1.6,v1.7,v1.8,v1.9,v1.11,v1.12,v1.13,next-candidate,"
```

Comment:
Include `NO-MILESTONE` in this list (instead of the empty string) to block PRs that don't have an associated milestone.

Reply:
added